test/runtime/Policies.go

```diff
@@ -1057,17 +1056,17 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 
 			By(fmt.Sprintf("Accessing %q on wrong port from %q should fail", dstIP, helpers.App1))
 			res = vm.ContainerExec(helpers.App1, helpers.CurlWithHTTPCode("http://%s:8080/public", dstIP))
-			ExpectWithOffset(1, res.WasSuccessful()).Should(BeFalse(),
+			ExpectWithOffset(1, res).Should(helpers.CMDSuccess(),
```

Comment:
Since this previously was `ExpectWithOffset(1, res.WasSuccessful()).Should(BeFalse()`, shouldn't it be `ExpectWithOffset(1, res).ShouldNot(helpers.CMDSuccess()` ?